index.html

```diff
@@ -301,5 +291,30 @@ <h2 class="section2-h2">skills</h2>
         </li>
       </ul>
     </section>
+
+    <!-- footer -->
+
+    <footer>
+      <h1 class="contact_title">Contact me</h1>
+      <p class="contact_paragraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form class="form-section">
```

Comment:
It would be better if you add a form `action` to collect user inputs. Collect data by using the [Formspree service](https://formspree.io/html). Also, add `method="Post"` to your form. These are all specified in the project requirements.  (see image below)
<img width="446" alt="Screenshot 2023-03-29 at 15 14 13" src="https://user-images.githubusercontent.com/107192076/228679949-93dbdd8f-589f-4083-adb9-530f02bb79f9.png">